x-pack/legacy/plugins/reporting/server/routes/generate_from_jobparams.ts

```diff
@@ -25,13 +26,45 @@ export function registerGenerate(
   server.route({
     path: `${BASE_GENERATE}/{exportType}`,
     method: 'POST',
-    config: getRouteConfig(request => request.params.exportType),
+    config: {
+      ...getRouteConfig(request => request.params.exportType),
```

Comment:
Nice I like this method better

Reply:
I have an unrelated small fix I can make to validate the exportType param is registered in the exportTypesRegistry as an ID of one of the types